src/ucs/memory/rcache.inl

```diff
@@ -14,7 +14,7 @@ ucs_rcache_region_test(ucs_rcache_region_t *region, int prot, size_t alignment)
 {
     return (region->flags & UCS_RCACHE_REGION_FLAG_REGISTERED) &&
            ucs_test_all_flags(region->prot, prot) &&
-           (region->alignment >= alignment);
+           ((alignment == 1) || (region->alignment >= alignment));
```

Comment:
why need special case for alignment == 1? is it just for optimization?
maybe pass alignment==0 and then compiler could deduce the condition is always true because region->alignment is unsigned?